README.md

```diff
@@ -0,0 +1,32 @@
+RateXBlock
+==========
+
+This is a basic clone of Dropthought for use in Open edX. This doesn't
+have all the analytics niceness, and the UX is less polished.
+
+I'm not positioning this as a competitor to DropThought or similar 
+services; ideally, once we collect the data, we'll just ship the data 
+off to their fancy analytics algorithms, but we won't have to bother
+with (a somewhat less reliable, integrated, and performant) 
+services-based integration, and we'll have something: 
+
+* Nicely integrated from a user standpoint
+* Data lives in the platform, so we can use our own analytics pipeline 
+  on it as well. 
+
+As of the time this README is written, we still need to apply styling,
```

Comment:
I think we'll finalize styling after the prototype pilot. Right now, it's usable Chrome+Ubuntu, Chrome+Firefox, and apparently a bit less so your OS. I agree we should update this, but I'd do it once we're bringing this into production (at which point, I'd like to pull in UX to help).